test/Scientist.Test/ScientistTests.cs

```diff
@@ -744,5 +744,85 @@ public void KeepingItClean()
 
             mock.Received().Clean(expectedResult);
         }
+
+        [Fact]
+        public void NoCandidate()
```

Comment:
I think the method name should communicate the expected outcome. For example, `NoCandidateStillRunsControl`.